src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -279,6 +279,7 @@ public void TimeOut_SetThenGet_ValuesMatch(Uri remoteServer)
             Assert.Equal(int.MaxValue, request.Timeout);
         }
 
+        [ActiveIssue(22627)]
```

Comment:
I'm a little confused.  The assert failure in the linked test shows the Actual value as being 0, and it's failing the assert because it's less than 1, not because it's greater than 15000.  Wouldn't the better fix be to change the Assert.InRange to have a lower bound of 0 rather than 1?

Reply:
That is a possible fix to the test.  I still need to research other instances of this test failing to see if that is the only reason.  It's possible that that network operation happens so fast that it can't timeout due to the granularity of the timeout being in milliseconds.

Reply:
For now, the test failures in CI is causing enough noise that it needs to be disabled right now.